sponsor.html

```diff
@@ -104,102 +106,152 @@ <h2> Let's Get In Touch </h2>
           <form>
             <div class="info-input">
               <div class="form-row">
-                <div class="form-group col-md-8">
+                <div class="form-group col-md-7">
                   <label class="sr-only" for="name-input">Full Name</label>
                   <input type="text" class="form-control" id="name-input" placeholder="Full Name">
                 </div>
-                <div class="form-group col-md-4">
+                <div class="form-group col-md-5">
                   <label class="sr-only" for="company-input">Company Name</label>
                   <input type="text" class="form-control" id="company-input" placeholder="Company Name">
                 </div>
               </div>
-              <div class="form-group">
+              <div class="form-group mb-0">
                 <label class="sr-only" for="sponsor-email-input">Email Address</label>
                 <input type="email" class="form-control" id="sponsor-email-input" placeholder="Email Address">
               </div>
             </div>
-            <div class="form-group">
+            <div class="form-group inquiry-box mb-1">
               <label class="sr-only" for="inquiry-input">Inquiry</label>
-              <input type="text" class="form-control" id="inquiry-input" placeholder="Inquiry">
+              <textarea class="form-control" id="inquiry-input" rows="7" placeholder="Inquiry"></textarea>
+            </div>
+            <div class="text-right">
+              <button type="submit" class="btn inquiry-btn">Send</button>
```

Comment:
In the mockups, the send button is inside of the text box. Can we move it inside so it conforms to the mocks?
![Screen_Shot_2020-04-02_at_10_31_29_PM](https://user-images.githubusercontent.com/30019505/78318149-0fac8300-7532-11ea-94d0-4c56de617ccc.png)

Reply:
lol honestly wasnt sure how to overlay it in the "inquiry" box :( i can do some google searching again tho